src/uint/modular/constant_mod.rs

```diff
@@ -176,6 +203,7 @@ impl<MOD: ResidueParams<LIMBS>, const LIMBS: usize> Zero for Residue<MOD, LIMBS>
 }
 
 #[cfg(feature = "rand_core")]
+#[allow(deprecated)]
```

Comment:
`new_checked()` can be used here

Reply:
I don't think it can be, without changing the trait definition in a breaking way.